pkg/types/library.go

```diff
@@ -1,5 +1,12 @@
 package types
 
+import "github.com/aquasecurity/trivy-db/pkg/db"
+
+type VersionInfo struct {
```

Comment:
`library.go` is the file for the programming language library. It is better to put this struct in another file.

Reply:
I moved it out to `app.go` in favour of temporal locality https://github.com/aquasecurity/trivy/pull/435/commits/de09a7fabd68652ef5bc54032e337c1967a3bdfd